paper/paper.tex

```diff
@@ -0,0 +1,946 @@
+\[
+f_*\mu(S) = \mu(f^{-1}(S))\ .
+\]
+
+In the following sections, we first discuss in the context of probability measures before the more general case.
+
+\subsection*{Forward parameterization}
+
+Starting with a $k$-dimensional multivariate random variable $z$, an \href{https://en.wikipedia.org/wiki/Affine_transformation}{\emph{affine transform}} is a ``linear transform with a shift''. We can use this to define a new random variable $x$, as
+\[
+\bX = \sigma \bZ + x_0\ ,
+\]
+
+with $\sigma$ and $x_0$ of the appropriate dimensions.\footnote{We would typically use $\mu$ in place of $x_0$, if not for the unfortunate potential confusion with $\mu$ as a name for a measure.} If $\EE[\bZ]=0$, then $x$ has mean
+\[
+\EE[\bX] = \sigma \EE[\bZ] + x_0 = x_0
+\]
+and variance matrix
+\[
+\VV[\bX] = \EE[(\bX-x_0)(\bX-x_0)^t] = \EE[\sigma \bZ \bZ^t \sigma^t] = \sigma \VV[\bZ] \sigma^t\ .
+\]
+
+Note that if $\VV[\bZ]=I_k$, we get $\VV[\bX]=\sigma \sigma^t$, so we can arrive at a given positive semidefinite $\VV[\bX]=\Sigma$ by taking $\sigma$ to be its \href{https://en.wikipedia.org/wiki/Cholesky_decomposition}{\emph{lower Cholesky factor}}. Also, in the special case of a one-dimensional Gaussian, this gives the familiar $\VV[\bX] = \sigma^2$.
+
+We call this the \emph{forward parameterization} because it's especially convenient for sampling, sometimes referred to as ``running the model forward''. Unfortunately, the cost of this convenience is a relatively awkward expression for the density. In this case, we start with $x$ and need to solve $z$, finally adjusting according to the determinant of the transformation:
+\[
+p_\bX(x) = \frac{1}{|\sigma|}\ p_\bZ(z) = \frac{1}{|\sigma|}\ p_\bZ\left(\sigma^{-1} (x - x_0)\right)\ ,
+\]
+
+where $|\sigma|$ is the \href{https://en.wikipedia.org/wiki/Determinant}{\emph{determinant}} of the square matrix, and the second equality comes from solving for $z$, which gives $z=\sigma^{-1}(x-x_0)$. More generally (when the transform is not expressed as a matrix), this role is played by the determinant of the \href{https://en.wikipedia.org/wiki/Jacobian_matrix_and_determinant}{\emph{Jacobian}}, $\left|\frac{\dd x}{\dd z}\right|$.
+
+This requires solving a linear system. Even with $\sigma$ being lower-triangular, this involves division operations and the allocation of a temporary vector for storage of $z$.
+
+In many cases, we prefer the density (or log-density, really) to be fast to evaluate. This leads us to a kind of dual approach to the above.
+
+\subsection*{Inverse parameterization}
+
+An alternative parameterization of a multivariate Gaussian is in terms of its \href{https://en.wikipedia.org/wiki/Precision_(statistics)}{\emph{precision matrix}}, $\Lambda = \VV[\bX]^{-1}$. Similarly to above, we'll write $\lambda$ for the lower Cholesky factor of $\Lambda$, so $\lambda \lambda^t = \Lambda$. The parameterization is then specified by
+\[
+\bZ = \lambda(\bX - \mu)\ .
```

Comment:
```suggestion
\bZ = \lambda\cdot(\bX - \mu)\ .
```

Reply:
why \cdot? This is matrix-vector multiplication

Reply:
I just thought it looks like a function application otherwise. For me \cdot is just multiplication, but perhaps we need to be more careful if we used it somewhere else?

Reply:
Hmm, I haven't thought of \cdot outside of vector-vector. In that case it involves an adjoint, so wouldn't it have an adjoint in general?

How about we add a `\<space>` and then a comment after that it's matrix-vector multiplication?

Reply:
just add \,